docker-compose-test.yml

```diff
@@ -38,9 +38,11 @@ services:
       - npm_cache:/app/node_modules
     command: /bin/bash -c "bundle && npm install && rake db:create db:migrate assets:precompile cucumber"
   db:
-    image: postgres:9.5
+    image: postgres:9.3.5
   selenium:
     image: selenium/standalone-chrome-debug
+    volumes:
+      - /dev/shm:/dev/shm
```

Comment:
What's this volume for?

Reply:
Shared memory for chrome - without this, chrome quickly runs out of shared memory and you get test failures - this is a known workaround

Reply:
Nice, thanks for the info